corehq/ex-submodules/pillowtop/management/commands/add_kafka_partition.py

```diff
@@ -29,7 +29,7 @@ def handle(self, topic, num_partitions, **options):
         )
         added_partition = raw_input("have you run {} ? [y/n]".format(kafka_command))
         if added_partition not in ['y', 'yes']:
-            print("then run it")
+            print("then run it on the kafka machine")
 
         for checkpoint in DjangoPillowCheckpoint.objects.filter(sequence_format='json'):
```

Comment:
yes. this will need to change i'll do that in a followup pr.

It shouldn't block running this command right now. I think the only error will be if we add a new kafka pillow (which we haven't so far)